packages/core/src/controllers/ConnectionController.ts

```diff
@@ -247,5 +252,48 @@ export const ConnectionController = {
     } catch (error) {
       throw new Error('Failed to disconnect')
     }
+  },
+
+  /**
+   * @experimental - This is an experimental feature and may be subject to change.
+   * Initializes SIWX if available.
+   * This is not yet considering One Click Auth.
+   */
+  async initializeSWIXIfAvailable() {
```

Comment:
I was following what I proposed in the [specs](https://www.notion.so/walletconnect/From-SIWE-to-SIWX-f409bc36c1214770aa4ecc947e7bee5c), but as it would be changes in two functions of this file I've created a single function for it. Would you recommend to place it somewhere else?